core/app/run.go

```diff
@@ -70,6 +70,8 @@ type VersionSpec struct {
 	Minor int
 	// Point version, not used if <0
 	Point int
+	// Dev version, not used if <0
+	Dev int
```

Comment:
Shouldn't this just be part of the build string?

Reply:
The point here is to be able to use this field in `func (v VersionSpec) GreaterThan(o VersionSpec) bool`

If we don't store this "dev" part of the version as a field, then the whole version comparison logic becomes convoluted as we would use `GreaterThan()` only for the regular part of the version, and then compare the "dev" version on the side.

Hence the addition of the "dev" version as a field.

Reply:
I got rid adding the "Dev" field to the Version